config/crd/bases/configuration.konghq.com_kongingresses.yaml

```diff
@@ -47,6 +47,39 @@ spec:
                 minimum: 0
                 type: integer
             type: object
+          eventhooks:
```

Comment:
KongIngress is a resource that allows the user of KIC to tune certain settings of the "_ingress_ abstraction" that are a superset of the `Ingress` resource in Kubernetes. For example, set a buffer size, or a more precise (than allowed by `Ingress`) request filter. There is a "zero..one" to "one" relationship between KongIngress and Ingress.

Event hooks, however, are entities that are completely disconnected from ingresses, and there is no relationship whatsoever between ingresses and event hooks.

Consider the following example: the user has KIC and a Kong Gateway with an empty configuration, has no Ingress resources, but wants to configure a `log` event handler.